### ISSUE TITLE
2.0 no longer compiles text LCD with progress bar

### ISSUE THREAD

**Issue by siana** [MarlinFirmware/Marlin#12415]
Compilation log excerpt:

```
sketch\src\lcd\HD44780\ultralcd_impl_HD44780.cpp: In static member function 'static void MarlinUI::draw_status_message(bool)':

sketch\src\lcd\HD44780\ultralcd_impl_HD44780.cpp:620:83: error: 'has_status' was not declared in this scope

     if (ELAPSED(millis(), progress_bar_ms + PROGRESS_BAR_MSG_TIME) || !has_status()) {
```
Presumably as a result of changeset a0c795b * Encapsulate common display code in a singleton (#12395)

Prerequisites:
#define REPRAP_DISCOUNT_SMART_CONTROLLER
#define LCD_PROGRESS_BAR

**Comment by dahuafschmied** [comment]
i have the same problem. 

- has_status()

does not exist in the project.

**Comment by thinkyhead** [comment]
Merged 54ec5528a0d8ebf4573af3ec78d8141d8a90dceb which … should … fix the issue.

**Comment by dahuafschmied** [comment]
fixed. Thank you
